AdblockPlusSafari/AdblockPlus.m

```diff
@@ -17,15 +17,15 @@
 
 #import "AdblockPlus.h"
 
-@import SafariServices;
-
-static NSString *AdblockPlusActivated = @"AdblockPlusActivated";
+NSString *AdblockPlusActivated = @"AdblockPlusActivated";
 static NSString *AdblockPlusEnabled = @"AdblockPlusEnabled";
 static NSString *AdblockPlusAcceptableAdsEnabled = @"AdblockPlusAcceptableAdsEnabled";
+static NSString *AdblockPlusFilterlists = @"AdblockPlusFilterlists";
```

Comment:
Nit: "filter list" being two words, the name here should be `AdblockPlusFilterLists` with an upper case L.